app.R

```diff
@@ -34,17 +39,116 @@ sass(
   output = "about/app.css"
 )
 
-# ??source
-sourceDir <- function(path, trace = TRUE, ...) {
-  for (nm in list.files(path, pattern = "[.][RrSsQq]$")) {
-    # if(trace) cat(nm,":")
-    source(file.path(path, nm), ...)
-    # if(trace) cat("\n")
+# function for app preprocessing ----
+
+app_prepocess <- function(m_reg, info_df, mwi, app_start = T){
```

Comment:
oops, typo: `app_preprocess`

Recommend commenting input outputs, for whenever you can get to it.

```
# functionalize app pre-processing to accommodate user-input data
# m_reg: measure registry, based on the metadata.xlsx
# info_df: processed measure information (measure metadata)
# mwi: 
# app_start = T:

# returns ....
```

Reply:
also, wonder if it might be helpful to split this and declaring all the colors, etc. into a separate script

Reply:
fixed the typo. will consider doing this in the future!